packages/react/src/select/root/SelectRoot.test.tsx

```diff
@@ -315,10 +315,6 @@ describe('<Select.Root />', () => {
       await user.click(trigger);
       expect(handleOpenChange.callCount).to.equal(1);
       expect(handleOpenChange.args[0][0]).to.equal(true);
-
-      await user.click(trigger);
-      expect(handleOpenChange.callCount).to.equal(2);
-      expect(handleOpenChange.args[1][0]).to.equal(false);
```

Comment:
I don't think this test is valid, because the trigger doesn't need to be the click target to dismiss it, any outside area works fine

Reply:
For touch it does require `outsidePressEvent: 'mousedown'`, which matches `Menu`

Reply:
> I don't think this test is valid, because the trigger doesn't need to be the click target to dismiss it, any outside area works fine

It's not that is not valid, but both clicking the trigger or any outside area should close it. Maybe we can actually test both of these two scenarios.

Reply:
This particular test fails due to `pointer-events: none;` style, so you can't click the trigger, only the area above it (which is `inert`, similar to a backdrop being present). The end result is the same though